### PR TITLE
Prevent GCal from sending update

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

This PR fixes the double sending emails & double booking events on Google calendar.

Please see the Loom in the thread for more context.

Fixes # (issue)

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 1]
```diff
@@ -1656,6 +1656,15 @@ async function handler(
...
     const updateManager = await eventManager.reschedule(evt, originalRescheduledBooking.uid);
+
+    //delete original rescheduled booking (no seats event)
+    if (!eventType.seatsPerTimeSlot) {
+      await prisma.booking.delete({
+        where: {
+          id: originalRescheduledBooking.id,
+        },
+      });
+    }
```
Comment (joeauyeung) [inline packages/features/bookings/lib/handleNewBooking.ts:1667]: I needed to borrow this from #7910

File: packages/app-store/googlecalendar/lib/CalendarService.ts [hunk 2]
```diff
@@ -133,6 +133,7 @@ export default class GoogleCalendarService implements Calendar {
           calendarId: selectedCalendar,
           requestBody: payload,
           conferenceDataVersion: 1,
+          sendUpdates: "none",
```
Comment (joeauyeung) [inline packages/app-store/googlecalendar/lib/CalendarService.ts:136]: https://developers.google.com/calendar/api/v3/reference/events/insert#parameters

File: packages/app-store/googlecalendar/lib/CalendarService.ts [hunk 3]
```diff
@@ -234,18 +235,18 @@ export default class GoogleCalendarService implements Calendar {
...
         },
         function (err, evt) {
           if (err) {
             console.error("There was an error contacting google calendar service: ", err);
-
             return reject(err);
           }
 
           if (evt && evt.data.id && evt.data.hangoutLink && event.location === MeetLocationType) {
+            console.log("This meet point triggers");
```
Comment (zomars) [inline packages/app-store/googlecalendar/lib/CalendarService.ts:249]:
```suggestion
```

**Review by zomars — APPROVED** [review]
Ship it

File: packages/app-store/googlecalendar/lib/CalendarService.ts (hunk 2 above)
Comment (alannnc) [reply packages/app-store/googlecalendar/lib/CalendarService.ts:136]: image

**Review by alannnc — APPROVED** [review]
Seems like the way to do it now.

File: packages/app-store/googlecalendar/lib/CalendarService.ts (hunk 2 above)
Comment (PeerRich) [reply packages/app-store/googlecalendar/lib/CalendarService.ts:136]: @joeauyeung @zomars @emrysal are we really fine with these sideeffects?

Comment (emrysal) [reply packages/app-store/googlecalendar/lib/CalendarService.ts:136]: Yes, these warnings exists for typical applications "Create a booking" - it's not for booking software that sends booking invites on its own like we do 🙏

**Review by emrysal — APPROVED** [review]

**Pull request merged**